pages/morse.js

```diff
@@ -71,6 +73,10 @@ export default function ButtonAppBar() {
                 <FiberManualRecordIcon />
                 <Typography variant="h5">tečka</Typography>
               </Button>
+              <Button variant="outlined" onClick={handle_backspace_click}>
```

Comment:
I prefer to have it as the last button. And maybe lower size of the whole button (not just icon) would be fine to distinguish it from other buttons. Maybe different color.